README.md

```diff
@@ -98,6 +98,20 @@ This additional prop can be useful for generating image URLs using the [@sanity/
 ### For Marks
 When using custom Vue components as mark serializers, all properties of the block object will be passed via [`v-bind`](https://vuejs.org/v2/api/#v-bind). **To access the data, define the corresponding [props](https://vuejs.org/v2/guide/components-props.html) in your component**. You can use [slots](https://vuejs.org/v2/guide/components-slots.html) (e.g. `this.$slots.default`) to access the mark text or content.
 
+#### Functional Components
+If your mark serializer is purely presentational (no reactive data, no lifecycle methods), you may want to use a [functional component](https://vuejs.org/v2/guide/render-function.html#Functional-Components) instead for better performance. When using functional components, you can access the properties on the `props` object. The mark text or content will available on the `children` array.
```

Comment:
```suggestion
If your mark serializer is purely presentational (no reactive data, no lifecycle methods), you may want to use a [functional component](https://vuejs.org/v2/guide/render-function.html#Functional-Components) instead for better performance. When using functional components, you can access the properties on the `props` object.
```
If I can go a little further 😄 In my opinion it will never be the scope of this component to handle children logic so I wouldn't even mention it.

Reply:
Yes, and the example is enough for understanding how to display the content.